src/clients/momento/commands/sorted_set_range.rs

```diff
@@ -0,0 +1,79 @@
+use super::*;
+
+/// Performs a range query on a sorted set, returning the specified range of
+/// elements. Currently, this only supports selecting by index (rank) and
+/// selects all members of the sorted set.
+pub async fn sorted_set_range(
+    client: &mut SimpleCacheClient,
+    config: &Config,
+    cache_name: &str,
+    request: workload::client::SortedSetRange,
+) -> std::result::Result<(), ResponseError> {
+    SORTED_SET_RANGE.increment();
+
+    let result = match (request.start, request.end) {
+        (None, None) => {
+            timeout(
+                config.client().unwrap().request_timeout(),
+                client.sorted_set_fetch_by_index(
+                    cache_name,
+                    &*request.key,
+                    momento::sorted_set::Order::Ascending,
+                    ..,
```

Comment:
Can you consolidate the 4 conditions by doing something like
```
request.start.unwrap_or_else(|| 0)..request.end.unwrap_or_else(|| -1)
```

Reply:
That would be nice, I think some changes need to be made in the momento crate to better express the selection range. Will follow up with a separate PR if I find a better way without making changes to the momento crate, or after that crate is updated.